package.json

```diff
@@ -65,7 +66,6 @@
     "@testing-library/jest-dom": "5.16.5",
     "@testing-library/react": "14.0.0",
     "@testing-library/user-event": "14.4.3",
-    "@tsd/typescript": "5.1.6",
```

Comment:
TSTyche installs versions of the `typescript` package to cache folder and loads it from there. This is how testing on different versions works.

There is no need to bump the version manually. By default, the latest version is used. In the CI script `latest` is specified. This always will always point to `latest` dist tag on NPM.

In a way TSTyche works as micro package manager. It fetches metadata from NPM from time to time. This means that it needs internet connection. That’s a limitation, but not a painful one. Because it is enough to install a particular version of TypeScript once.